app/assets/javascripts/student_profile/record_service.js

```diff
@@ -197,7 +197,7 @@
           onClick: this.onClickCancel
         }, 'Cancel'),
         (this.props.requestState === 'pending') ? dom.span({}, 'Saving...') : null,
-        (this.props.requestState === 'error') ? dom.span({}, 'Try again!') : null
+        dom.span({}, this.props.requestState)
```

Comment:
Oh yeah, good point. Since the string isn't 'error' anymore, I guess I need a smarter way of knowing when to display it. It's not like I can just say requestState != 'pending'.